portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,23 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+
+/**
+ * Adds a random fact to the page.
+ 
+
+function addRandomGreeting() {
+  const facts = 
+     ['I am left handed', 'I used to play the piano', 'I go to Barnard College in NYC', 'I major in CS and minor in History and Architecte'];
+
+  // Pick a random fact
+  const fact = facts[Math.floor(Math.random() * facts.length)];
+
+  // Add it to the page
+  const factContainer = document.getElementbyId('greeting-container');
+  factContainer.innerText=fact;
+}
+
+*/
```

Comment:
This is commented out. If you're not planning to delete it, it's a good practice to delete it.